### ISSUE TITLE
isContract will always return true

### ISSUE THREAD

**Issue by tymat** [Dexaran/ERC23-tokens#2]
The `extcodesize(address)` will return `1` when called against an externally owned account.  

I modified isContract() to return the value of length.

```
truffle(test)> InterfaceAssetVendor.at(InterfaceAssetVendor.address).isContract.call(InterfaceAssetVendor.address)
[ true, { [String: '7333'] s: 1, e: 3, c: [ 7333 ] } ]
truffle(test)> InterfaceAssetVendor.at(InterfaceAssetVendor.address).isContract.call('0x94a69c2543edbb1efef8b6994bc8d0d5b5ffa641')
[ true, { [String: '1'] s: 1, e: 0, c: [ 1 ] } ]
```

**Comment by Dexaran** [comment]
> isContract will always return true

I don't understand what are you talking about.
`isContract` must return `true` if a given address is a contract and `false` if a given address is a wallet.

I wrote this small example:
` contract ReturnTest {`
`     bool public extCodeReturn=true;`
`     function isContractTest(address _addr) {`
`      uint length;`
`      assembly { length := extcodesize(_addr) }`
`        if(length>0) {`
`            extCodeReturn=true; `
`        }`
`        else { extCodeReturn=false; } } }`

I deployed it here on Ropsten: 0x8102d6c0CAc954BE65E60Eae602713c0c6e66f5D 
When I'm executing `isContractTest` with 0x8102d6c0CAc954BE65E60Eae602713c0c6e66f5D address, it makes `extCodeReturn = true`.
When I'm executing `isContractTest` with 0x00b6fa39793d0537c111df8f6968204f152b4515 (my Ropsten address) it makes `extCodeReturn = false`. It looks like everything is ok.
